src/Symfony/Component/String/Uuid.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class Uuid
+{
+    public const TYPE_1 = UUID_TYPE_TIME;
+    public const TYPE_3 = UUID_TYPE_MD5;
+    public const TYPE_4 = UUID_TYPE_RANDOM;
+    public const TYPE_5 = UUID_TYPE_SHA1;
+
+    private $uuid;
+
+    public function __construct(string $uuid)
+    {
+        if (!\function_exists('uuid_is_valid')) {
+            throw new \RuntimeException('The UUID extension is required. You can run "composer require symfony/polyfill-uuid".');
```

Comment:
You cannot use the "%s" class as the UUID extension is not installed. Try running [...]